src/modules/language/actions/setLanguageChangeUrls.js

```diff
@@ -1,5 +1,17 @@
 import { createAction } from 'redux-actions'
 
 export const setLanguageChangeUrls = createAction('SET_LANGUAGE_CHANGE_URLS',
-  (languageChangeUrls) => (languageChangeUrls)
-)
+  (mapLanguageToUrl, languages, availableLanguages = undefined) => (
```

Comment:
This is the same as:

```
 (mapLanguageToUrl, languages, availableLanguages = undefined) => {
    let a = (language) => mapLanguageToUrl(language.code, availableLanguages[language.code])
    let b = (language) => mapLanguageToUrl(language.code, undefined)

    return availableLanguages
      // languageChange of a specific page/event with ids in availableLanguages
      ? languages.reduce((accumulator, language) => {
        return ({
          ...accumulator,
          [language.code]: a(language)
        })
      }, {})
      : languages.reduce((accumulator, language) => {
        return ({
          ...accumulator,
          [language.code]: b(language)
        })
      }, {})
  }
```

Which is the same as:
```
(mapLanguageToUrl, languages, availableLanguages = undefined) => {
    let languageMapper
    if (availableLanguages) {
      languageMapper = (language) => mapLanguageToUrl(language.code, availableLanguages[language.code])
    } else {
      languageMapper = (language) => mapLanguageToUrl(language.code, undefined)
    }

    return languages.reduce((accumulator, language) => ({
      ...accumulator,
      [language.code]: languageMapper(language)
    }), {})
```

Language changing still works, so I'm pretty sure this is correct.

Reply:
Which can be simplified to:
```
(mapLanguageToUrl, languages, availableLanguages = {}) => {
    const languageMapper = (language) => mapLanguageToUrl(language.code, availableLanguages[language.code])

    return languages.reduce((accumulator, language) => ({
      ...accumulator,
      [language.code]: languageMapper(language)
    }), {})
  }
```

Because of the simplification in EventsPage.js

Reply:
If availableLanguages = undefined -> availableLanguages[language.code] = undefined

Reply:
done, good suggestions